scene/main/node.cpp

```diff
@@ -994,8 +994,28 @@ void Node::_validate_child_name(Node *p_child, bool p_force_human_readable) {
 
 		if (!unique) {
 			ERR_FAIL_COND(!node_hrcr_count.ref());
-			String name = "@" + String(p_child->get_name()) + "@" + itos(node_hrcr_count.get());
```

Comment:
I would keep this as a comment to make it clear what is going on in the rest of this block

Reply:
Good point, will add.